### PR TITLE
docs: update WebDriver BiDi documentation

### PR DESCRIPTION
**Pull request by jecfish**
- include Firefox announcement in the doc
- polishing

### CODE REVIEW

File: docs/webdriver-bidi.md [hunk 1]
```diff
@@ -1,57 +1,46 @@
 # Experimental WebDriver BiDi Support
```
Comment (jecfish) [inline docs/webdriver-bidi.md]:
```suggestion
# Experimental WebDriver BiDi support
```

**Comment by jecfish** [comment]
@mathiasbynens PTAL

File: docs/webdriver-bidi.md [hunk 2]
```diff
@@ -1,57 +1,46 @@
-# Experimental WebDriver BiDi Support
+# Experimental WebDriver BiDi support
 
 [WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) is a new cross-browser
-automation protocol that adds browser-driven events to WebDriver. Here are the
-resources if you want to learn more about WebDriver BiDi:
+automation protocol currently under development, aiming to combine the best of both WebDriver “Classic” and CDP. WebDriver BiDi promises bi-directional communication, making it fast by default, and it comes packed with low-level control. Dive deeper with these resources:
```
Comment (jrandolf) [inline docs/webdriver-bidi.md]:
```suggestion
automation protocol currently under development, aiming to combine the best of both WebDriver “Classic” and CDP. WebDriver BiDi promises bi-directional communication, making it fast by default, and it comes packed with low-level control.

See also:
```

**Review by jrandolf-google — APPROVED** [review]
One comment, but everything else is fine.

**Review by mathiasbynens — APPROVED** [review]
Thanks!

**Pull request merged**